sql-db/multiple-pus/src/main/resources/import-fruits.sql

```diff
@@ -0,0 +1,8 @@
+INSERT INTO fruit (id, name) VALUES (1, 'Apple');
+INSERT INTO fruit (id, name) VALUES (2, 'Pear');
+INSERT INTO fruit (id, name) VALUES (3, 'Strawberry');
+INSERT INTO fruit (id, name) VALUES (4, 'Plum');
+INSERT INTO fruit (id, name) VALUES (5, 'Cherry');
+INSERT INTO fruit (id, name) VALUES (6, 'Berry');
+INSERT INTO fruit (id, name) VALUES (7, 'Cranberry');
+UPDATE hibernate_sequence SET next_val = 8;
```

Comment:
Can we use `nextval` in the insert statements?

Reply:
No, `mariadb` doesn't like it much